src/release/breaking-changes/route-navigator-refactoring.md

```diff
@@ -88,16 +88,16 @@ MaterialApp(
 ```
 
 There are different ways to migrate this change.
-One way is to set the initial route name
-to a fixed value and generate a specific route
-(`FakeSplashRoute` in the above example)
-for the route name.
+One way is to set an explicit value for `MaterialApp.initialRoute`.
+You can then test for this value in place of `isInitialRoute`.
+As `initialRoute` inherits its default value outside of Flutter's scope,
+you must set an explicit value for it.
 
 ```dart
 MaterialApp(
-  initialRouteName: 'fakeSplash',
+  initialRoute: '/', // default value cannot be guaranteed at runtime
```

Comment:
This reads as if it's a warning about the value you pass in as `initialRoute`, but it seems the intention is to explain why an explicit `initialRoute` is needed in this situation. Explain the goal of the line more explicitly. Perhaps something like: "Explicitly set the initial route, as its default value can be manipulated."?